### PR TITLE
Abraxas customer story

### PR DESCRIPTION
**Pull request by charger28**
Abraxas customer story

### CODE REVIEW

**Comment by maxandersen** [comment]
https://quarkusio-quarkusio-github-io-preview-pr-1014.surge.sh/blog/ is rendering without icon/title - investigating.

File: _posts/2021-06-22-abraxas-customer-story.adoc [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: 'Java and Taxes: Two Things That Are Certain in a Developer's Life'
```
Comment (maxandersen) [inline _posts/2021-06-22-abraxas-customer-story.adoc:3]: ' inside '' messes up asciidoc parser.

```suggestion
title: "Java and Taxes: Two Things That Are Certain in a Developer's Life"
```

**Comment by maxandersen** [comment]
looks good to me - if i hear nothing else i'll push this out tomorow in 9-10 CET time frame.

**Comment by charger28** [comment]
Thanks for finding this issue. I must have missed this. It's good to go.

**Pull request merged**